graylog2-server/src/test/java/org/graylog/testing/completebackend/apis/Users.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.testing.completebackend.apis;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.restassured.path.json.JsonPath;
+import io.restassured.specification.RequestSpecification;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+
+public class Users implements GraylogRestApi {
+
+    private final RequestSpecification requestSpecification;
+
+    public Users(RequestSpecification requestSpecification) {
+        this.requestSpecification = requestSpecification;
+    }
+
+    public record User(@JsonProperty("username") String username,
```

Comment:
A quick thought : can't we reuse org.graylog2.rest.models.users.requests.CreateUserRequest class here?